torch/nn/attention/__init__.py

```diff
@@ -31,3 +39,37 @@ def _raise_kernel_warnings(params: SDPAParams) -> None:
         if not can_use_flash_attention(params):
             warn("Flash attention can't be used because:")
             can_use_flash_attention(params, True)
+
+
+@contextlib.contextmanager
+def sdpa_kernel(backend: Union[List[SDPBackend], SDPBackend]):
```

Comment:
should this be `backneds`?

Reply:
SDPBackends instead of SDPBackend?

Reply:
offline: change name to `backends` and only support List